main.js

```diff
@@ -0,0 +1,99 @@
+const renderer = new Renderer()
+const boardGame = new playerMatrix()
+
+$("#load-board").on("click", function () {
+    boardGame.playersScore.player1Score=0
+    boardGame.playersScore.player2Score=0
+    const rows = $("#rows").val()
+    const columns = $("#columns").val()
+    $("#rows").val("")
+    $("#columns").val("")
+    const coins = Math.floor((rows*columns)/5)
+    const walls = Math.floor((rows*columns)/10)
+    boardGame.loadBoard(rows, columns)
+    boardGame.player1(0, 0, 1)
+    boardGame.player2(rows, columns, 2)
+    boardGame.addingCoins(rows, columns, coins)
+    boardGame.addWalls(rows, columns, walls)
+    $(".board").css("grid-template-columns", `repeat(${rows}, 1fr)`)
+    renderer.renderBoard(boardGame.matrix)
+    renderer.renderScores(boardGame.playersScore)
+    // boardGame.print()
+    moveplayers()
+})
+
+const moveplayers = function () {
+    $(document).keypress(function (e) {
```

Comment:
the event listeners (in this case keypress) should not be inside a function but out in the global scope - there's no need for a function to wrap them in this case